README.md

```diff
@@ -1434,6 +1435,16 @@ Flags:
 
 Unimplemented `docker-compose down` (V1) flags: `--rmi`, `--remove-orphans`, `--timeout`
 
+### :whale: nerdctl compose stop
+
+Stop containers in services without removing them.
+
+Usage: `nerdctl compose stop [OPTIONS] [SERVICE...]`
+
+Flags:
+
+- :whale: `-t, --timeout`: Seconds to wait for stop before killing it (default 10)
+
 ### :whale: nerdctl compose ps
 List containers of services
 
```

Comment:
Could you also remove `nerdctl compose stop` from the unimplemented command list? https://github.com/containerd/nerdctl#unimplemented-docker-commands

Reply:
removed